### PR TITLE
1.19.4 - Simplified tab (Effects) + Persistent data (PRE-RELEASE)

### PR DESCRIPTION
**Pull request by Ascynx**
Made the mod auto-disable discord RPC if the computer's architecture is unsupported.

Added a new way to detect changing shards.

Added persistent data
	- Chest counter data is stored in between sessions (and used, if you have not changed shard in between mod usage)
	- Stores weekly bounty data

Migrated delve bounty handling to its own class
Migrated location notifications to its own new standalone notification system.

Refactored calculator in the form of a sub-element (Might fix a few issues tied to the old injection method).

Effect packets

### CODE REVIEW

**Comment by Ascynx** [comment]
this pr would close #27 and #28

File: src/main/java/ch/njol/unofficialmonumentamod/UnofficialMonumentaModClient.java [hunk 1]
```diff
@@ -124,9 +156,10 @@ public void onInitializeClient() {
 
 	public static void onDisconnect() {
 		abilityHandler.onDisconnect();
-		Notifier.onDisconnect();
 		LocationNotifier.onDisconnect();
 		spoofer.onDisconnect();
+		//TODO if false, re-schedule unload.
```
Comment (Njol) [inline src/main/java/ch/njol/unofficialmonumentamod/UnofficialMonumentaModClient.java]: Does this TODO still need to be addressed?

**Review by Njol — APPROVED** [review]
Love the new effects handling with packets!
I only found minor formatting issues that i didn't bother commenting on, and a stray TODO that may still need handling.

File: src/main/java/ch/njol/unofficialmonumentamod/UnofficialMonumentaModClient.java (hunk 1 above)
Comment (Ascynx) [reply src/main/java/ch/njol/unofficialmonumentamod/UnofficialMonumentaModClient.java]: indeed.

**Pull request merged**